DQM/TrackingMonitorSource/interface/HLTPathSelector.h

```diff
@@ -0,0 +1,41 @@
+#ifndef DQM_TrackingMonitorSource_HLTPathSelector_h
+#define DQM_TrackingMonitorSource_HLTPathSelector_h
+
+#include <vector>
+#include <string>
+#include <map>
+#include "TPRegexp.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EDFilter.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "HLTrigger/HLTcore/interface/HLTConfigProvider.h"
+#include "DataFormats/Common/interface/TriggerResults.h"
+#include "DataFormats/HLTReco/interface/TriggerEvent.h"
+
+class HLTPathSelector : public edm::EDFilter {
+public:
+  explicit HLTPathSelector(const edm::ParameterSet&);
+  virtual ~HLTPathSelector();
+
+  void beginRun(edm::Run const &, edm::EventSetup const&);
+  bool filter(edm::Event&, edm::EventSetup const&);
+  void endJob();
+
```

Comment:
Please add the ```override``` keyword to the above functions. That way the complier will guarantee there are no typos.
The same holds for all other modules in the pull request.